docs/core/project-sdk/msbuild-props.md

```diff
@@ -266,6 +266,18 @@ The `PreserveCompilationReferences` property is similar to the [PreserveCompilat
 
 For more information, see [Razor SDK properties](/aspnet/core/razor-pages/sdk#properties).
 
+### ProduceReferenceAssemblyInOutDir
+
+The `ProduceReferenceAssemblyInOutDir` property controls whether reference assemblies are written to the `OutDir` directory. Staring in .NET 6, the default value is `false`, and reference assemblies are only written to the `IntermediateOutputPath` directory. Set the value to `true` to write reference assemblies to the `OutDir` directory.
```

Comment:
```suggestion
The `ProduceReferenceAssemblyInOutDir` property controls whether reference assemblies are written to the `OutDir` directory. Starting in .NET 6, the default value is `false`, and reference assemblies are only written to the `IntermediateOutputPath` directory. Set the value to `true` to write reference assemblies to the `OutDir` directory.
```

Reply:
Was this property only introduced in .NET 6?

Reply:
Seems like! I jumped into the msbuild code history and it was added ~2 months ago in the same commit that the ref assemblies change was made.